src/views/home/post/HTMLPost.vue

```diff
@@ -7,7 +7,7 @@ import { Component, Vue, Prop } from "vue-property-decorator";
 import { Article } from "activitypub-objects";
 
 @Component
-export default class ActivityStreamsArticleComponent extends Vue {
+export default class HTMLPostComponent extends Vue {
```

Comment:
Sure you don't want to name the file and class `->Html<-Post[Component]`?

Reply:
I'm not happy with the name. Would ActivityStreamsArticleNote be better? Maybe you have a better one?
Component can be left out, because I used the Angular naming convention

Reply:
How about:

```
class ActivityStreamsType extends Vue;
class ActivityStreamsTextType extends Vue;

class ActivityStreamsArticleType extends ActivityStreamsTextType;
class ActivityStreamsNoteType extends ActivityStreamsTextType;
```